hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1523,39 +1521,6 @@ public void testGetTrashRoots() throws IOException {
     Assert.assertTrue(volume1.setOwner(prevOwner));
   }
 
-  /**
-   * Check that  files are moved to trash since it is enabled by
-   * fs.rename(src, dst, options).
-   */
-  @Test
-  @Flaky({"HDDS-5819", "HDDS-6451"})
```

Comment:
This test is flaky as it checks for existence of key in /Current but the checkpoint interval is in milliseconds so it might already be in checkpoint directory. This part is already tested in testTrash , hence removing this test.

Reply:
thanks for the info!